dev-test/minio-operator/subscription.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: minio-operator
+  namespace: openshift-operators
```

Comment:
@faizanahmad055 we should create separate namespace named minio-operator and deploy to it; we are discouraging deploying everything to openshift-operators